src/client/components/pages/Courses/__tests__/MeetingModal.test.tsx

```diff
@@ -933,6 +933,88 @@ describe('Meeting Modal', function () {
             });
           });
         });
+        describe('On Delete Behavior', function () {
+          context('when a meeting time is being edited', function () {
+            const cs50FridayMeeting = testCourseInstance[semKey].meetings
+              .filter((meeting) => meeting.day === DAY.FRI)[0];
+            const cs50TuesdayMeeting = testCourseInstance[semKey].meetings
+              .filter((meeting) => meeting.day === DAY.TUE)[0];
+            beforeEach(async function () {
+              const editCS50FridayMeetingButton = await waitForElement(() => document.getElementById('editMeetingButton' + cs50FridayMeeting.id));
```

Comment:
Instead of using `document.getElementById`, this should be using something `findByLabelText(/Edit Meeting \d on Friday/)` and querying on the `alt` prop text for the delete button`.

Reply:
This was fixed in d26877b.